int64 amount_of_drones
char  shape_type
float64 distance
float64 angle
float64 random_range
---
int64 matrix_size
float64[] connection_matrix
float64[] start_pose